kvmagent/kvmagent/plugins/vm_local_volume_cache/command_wrapper/qemu_img.py

```
@@ -0,0 +1,362 @@
+        return base_url
+
+    def __init__(self, volume, addons):
+        super().__init__(volume, addons)
+        self.__parse_iscsi_url()
+    
+    def __parse_iscsi_url(self):
+        assert self.volume, "volume must be set"
+        assert self.volume.installPath, "volume.installPath must be set"
+
+        url = self.volume.installPath # type: str
+        portal, self.target, self.lun = url.lstrip("iscsi://").split("/")
```

Ruff (0.15.2)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)